tools/compiler/src/main/java/org/apache/cxf/tools/compiler/Compiler.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.apache.cxf.common.util;
```

Comment:
Could we please keep the same package? I understand that we would have package split brain but this is public API and if we want this change into 3.4.x / 3.5.x / 3.6.x, we should not break the possible consumers.

Alternatively, we could keep package change BUT introduce `org.apache.cxf.common.util.Compiler` class which will delegate to `org.apache.cxf.tools.compiler.Compiler`: it seems like public API is not using any dependencies from `java.compiler` APIs.